src/frontend/pages/home/home.jsx

```diff
@@ -1,9 +1,93 @@
+            </div>
+          </div>
+        </div>
+
+        <div className="home-body">
+        <div className="deal-container flex-col ">
+          <p className="deal-heading flex center">BEST DEALS</p>
+          <div className="brand-cards flex ">
+            <div className="card">
+              <div className="body">
+                <div className="img-div">
+                  <img className="deal-card-img-size" src={require('../../assets/laptop.png')}alt="samsung"/>
+                </div>
+        
+                <div className="body-content div-padding">
+                  <h3 className="card-title" onClick={()=> Navigate('/products')}>Thin & Light Laptop</h3>
+                  <h4 className="card-sub-title">Reimagine the Way Your Kids Learn No Cost EMI & more.</h4>
+                </div>
+              </div>             
+            </div>
+
+            <div className="card">
+              <div className="body">
+                <div className="img-div">
+                  <img className="deal-card-img-size" src={require('../../assets/laptop.png')}alt="samsung"/>
+                </div>
+        
+                <div className="body-content div-padding">
+                  <h3 className="card-title" onClick={()=> Navigate('/products')}>Budget Laptop </h3>
+                  <h4 className="card-sub-title">Learning Made Engaging and Fun. Exclusive Deals with Great Offers.</h4>
+                </div>
+              </div>             
+            </div>
+
+            <div className="card">
+              <div className="body">
+                <div className="img-div">
+                  <img className="deal-card-img-size" src={require('../../assets/laptop.png')}alt="samsung"/>
+                </div>
+        
+                <div className="body-content div-padding">
+                  <h3 className="card-title" onClick={()=> Navigate('/products')}>Gaming Laptop</h3>
+                  <h4 className="card-sub-title">Powerful gaming setups for brilliant wins.</h4>
+                </div>
+              </div>
+            </div>
+                  
+          </div>
+        </div>
 
-    <h1> Home page </h1>
+        <div>
+          <p className="deal-heading center flex">Featured Brands</p>
+          <div className="deal-container flex center">
+            <div className="brand">
+                <img className="brandLogo" src={require('../../assets/dell-144.png')} alt="dell" />
+              </div>
+              
+              <div className="brand">
+                  <img  className="brandLogo" src={require('../../assets/hp-144.png')} alt="hp" />
+              </div>
+              <div className="brand">
+                  <img className="brandLogo" src={require('../../assets/samsung-144.png')}alt="samsung"/>
+              </div>
+              <div className="brand"> 
+              <img className="brandLogo" src={require('../../assets/asus-144.png')} alt="asus"/>
```

Comment:
You should write all the import/require statements at the start of your code.
Bring these lines to the top, you can store them in variables and use them here.

Reply:
yeah thanks never thought about this. i have stored theme in variable and used later